R/phylo_check.R

```diff
@@ -0,0 +1,133 @@
+#' Check phylogeny tip names
+#'
+#' A function to check the list of tip names in a phylogeny against a vector of
+#' taxon names, and if desired, to trim the phylogeny to only include taxon
+#' names within the vector.
+#'
+#' @param tree \code{phylo}. A phylo object. Phylogenies can be read into R from
+#' .txt or .tree files containing the Newick formatted tree using
+#' \code{ape::read.tree}.
```

Comment:
Part of me thinks the extra details about reading phylogenies should be put in a "Details" section.